src/slack.rs

```diff
@@ -147,9 +147,23 @@ async fn handle_disconnect_message(
 async fn handle_slash_command(
     message: incoming::Incoming<incoming::SlashCommandIncomingMessage>,
 ) -> outgoing::SlackOutgoingMessage {
-    match message.payload.command.as_str() {
-        "/spin" => {
-            let pizza = get_random_pizza();
+    let command_spin_mode = match message.payload.command.as_str() {
+        "/spin" => Some(SpinMode::Any),
+        "/spin-vegan" => Some(SpinMode::Vegan),
+        "/spin-vegetarian" => Some(SpinMode::Vegetarian),
+        _ => None,
```

Comment:
I would much rather early exit here, than return None and handle the case below. That way you can have spin_command_mode: SpinMode